pkgs/development/libraries/duckdb/default.nix

```diff
@@ -106,10 +106,12 @@ stdenv.mkDerivation rec {
     '';
 
   meta = with lib; {
-    homepage = "https://github.com/duckdb/duckdb";
+    changelog = "https://github.com/duckdb/duckdb/releases/tag/v${version}";
     description = "Embeddable SQL OLAP Database Management System";
+    homepage = "https://duckdb.org/";
     license = licenses.mit;
-    platforms = platforms.all;
+    mainProgram = "duckdb";
     maintainers = with maintainers; [ costrouc cpcloud ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```
Defaults to `python.meta.platforms`, which is more reasonable than `platforms.all`.

Reply:
this is the `stdenv.mkDerivation` and if i am reading the code correctly it looks like the default is [lib.platforms.all](https://github.com/NixOS/nixpkgs/blob/e6910fd16e77f1f5f15cd5e22d725d6afd6ef5ee/pkgs/stdenv/generic/default.nix#L138). The python module is as you suggest, there is no platform set in the meta attribute.

Reply:
My bad.